Source/MicrowaveNetworks/Touchstone/IO/TouchstoneWriter.cs

```diff
@@ -118,7 +118,11 @@ private static string FormatOptions(TouchstoneOptions options)
             string frequencyUnit = TouchstoneEnumMap<FrequencyUnit>.ToTouchstoneValue(options.FrequencyUnit);
             string parameter = TouchstoneEnumMap<ParameterType>.ToTouchstoneValue(options.Parameter);
             string format = TouchstoneEnumMap<FormatType>.ToTouchstoneValue(options.Format);
-            string resistance = $"{ResistanceChar} {options.Resistance:g}";
+            string sign = options.Reactance >= 0 ? "+" : "";
```

Comment:
~~Shouldn't the false case be '-'?~~
``` c#
string sign = options.Reactance >= 0 ? "+" : "-";
```

Never mind, I see now. Perhaps a comment on this line to point out that the negative sign will come automatically from converting the negative number to text?